ferrischat_webserver/src/invites/use_invite.rs

```diff
@@ -0,0 +1,109 @@
+use actix_web::{HttpRequest, HttpResponse, Responder};
+use ferrischat_common::types::InternalServerErrorJson;
+use time;
+
+pub async fn use_invite(req: HttpRequest, auth: crate::Authorization) -> impl Responder {
+    let invite_code = get_item_id!(req, "code");
+
+    let user_id = auth.0;
+    let bigint_user_id = u128_to_bigdecimal(user_id);
+
+    let db = get_db_or_fail!();
+
+    let resp = sqlx::query!("SELECT * FROM invites WHERE code = $1", invite_code)
+        .fetch_optional(db)
+        .await;
+
+    match resp {
+        Ok(resp) => match resp {
+            Some(invite) => {
+                let uses = invite.uses + 1;
+                let now = {
+                    let now = time::OffsetDateTime::now_utc();
+                    time::PrimitiveDateTime::new(now.clone().date(), now.time())
+                };
+                if invite.max_uses < uses {
```

Comment:
I think it'd make more sense to swap the comparison statement here, I was slightly confused when looking at this at first.
```suggestion
                if uses > invite.max_uses {
```